chapters/sampling-randomization-power.tex

```diff
@@ -1,391 +1,443 @@
-This type of structure is called \textbf{clustering},\sidenote{\url{https://dimewiki.worldbank.org/wiki/Multi-stage_(Cluster)_Sampling}}
-because the units are assigned to treatment in clusters.
-Because the treatments are assigned in clusters, however,
-there are in reality fewer randomized groups than the number of units in the data.
-Therefore, standard errors for clustered designs must also be clustered,
-at the level at which the treatment was assigned.\sidenote{\url{https://blogs.worldbank.org/impactevaluations/when-should-you-cluster-standard-errors-new-wisdom-econometrics-oracle}}
-
-Clustered randomization must typically be implemented manually;
-it typically relies on subsetting the data intelligently
-to the desired assignment levels.
-We demonstrate here.
-
-\codeexample{randtreat-clusters.do}{./code/randtreat-clusters.do}
-
-%------------------------------------------------
-\section{Power calculations}
-
-When we have decided on a practical sampling and randomization design,
-we next assess its \textbf{power}.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Power_Calculations_in_Stata}}
-\index{power}
-Statistical power can be described in a few ways,
-each of which has different uses.\sidenote{\url{http://www.stat.columbia.edu/~gelman/stuff_for_blog/chap20.pdf}}
-The purpose of power calculations is not to
-demonstrate that a study is ``strong'',
-but rather to identify where the strengths and weaknesses
-of your design are located, so that readers
-can correctly assess the evidentiary value of
-any results (or null results) in the analysis.
-This should be done before going to the field,
-across the entire range of research questions
-your study might try to answer,
-so you know the relative tradeoffs you will face
-by changing your sampling and randomization schemes
-and can select your final strategy appropriately.
-
-The classic definition of power is
-``the likelihood that your design detects a significant treatment effect,
-given that there is a non-zero true effect in reality''.
-Here we will look at two useful practical applications
-of that definition and show what quantitative results can be obtained.
-We suggest doing all power calculations by simulation;
-you are very unlikely to be able to determine analytically
-the power of your study unless you have a very simple design.
-Stata has some commands that can calculate power for
+but the study is interested in household behavior.
+This type of structure is called \textbf{clustering},\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Multi-stage_(Cluster)_Sampling}}
+and the groups in which units are assigned to treatment are called clusters.
+The same principle extends to sampling:
+it may be infeasible to decide whether to test individual children
+within a single classroom, for example.
+
+Clustering is procedurally straightforward in Stata,
+although it typically needs to be performed manually.
+To cluster a sampling or randomization,
```

Comment:
randomization is typically clustered by states, villages, schools, classrooms (teachers), clinics (doctors), districts (politicians), courtrooms (judges), households, whenever that is the level at which the policy is to be implemented. by contrast the observations are often children, patients, cases, etc. so you have to cluster at the level the induced variation occurs: the randomization unit. otherwise you are massively overestimating the power of the experiment and the CIs are waaaaay too narrow.

ie if I have 2 doctors and I train 1 of them, I could say I did a really strong experiment by assessing them over 1,000 patients each. but that's not true: it's really N=2 in randomization and the cluster estimator does exactly that adjustment -- in this extreme case it should be basically non-estimable!

Reply:
This can happen, say you are doing an educational intervention so you randomize over schools but deliver programs to children, or you are doing ag, you randomize over villages and give things to farmers. I think you are describing the same idea with a slightly different language choice?